mobile/android/app/build.gradle

```diff
@@ -135,11 +135,11 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
     defaultConfig {
-        applicationId "com.icard"
+        applicationId 'com.llamoss0501.icard'
```

Comment:
do you know why there is llamos0501 in the app id and in other places in the android folder

Reply:
This is because I'm trying to build an app using my expo account. I will research how to fix this. We can make an organization so that our app will have a packaging distributor named after our team (uofablueprint for example).

Reply:
Yes, we should definitely do that. I'll merge this ticket for now and we can do that in the new expo issue